File: js/lv1/문자열 다루기 기본.js
//[level 1] 문자열 다루기 기본 12918
//https://school.programmers.co.kr/learn/courses/30/lessons/12918

//결과 
//정확성: 100.0
//합계: 100.0 / 100.0

function solution(s) {
    if (s.length == 4 || s.length == 6) {
        return s.split("").every(c => !isNaN(c))
    } else return false;
}

File: py/Level1/문자열 다루기 기본.py
#
# [level 1] 문자열 다루기 기본
# https://school.programmers.co.kr/learn/courses/30/lessons/12918

# 결과
# 정확성: 100.0
# 합계: 100.0 / 100.0

def solution(s):
    if len(s) == 4 or len(s) == 6:
        return s.isdigit()
    else: return False